tests/bluetooth/audio/bap_broadcast_source/src/main.c

```diff
@@ -170,6 +171,10 @@ ZTEST_F(bap_broadcast_source_test_suite, test_broadcast_source_create_delete)
 	err = bt_bap_broadcast_source_create(create_param, &fixture->source);
 	zassert_equal(0, err, "Unable to create broadcast source: err %d", err);
 
+	zassert_equal(create_param->qos->sdu, stream->qos->sdu, "Unexpected stream SDU");
+	zassert_equal(create_param->qos->rtn, stream->qos->rtn, "Unexpected stream RTN");
+	zassert_equal(create_param->qos->phy, stream->qos->phy, "Unexpected stream PHY");
```

Comment:
I'm not really sure why you have these asserts here.
´stream´ is fetched from ´create_param´, and between the assignment and these asserts I cannot see anything that could actually change ´create_param´ or its members.

Wouldn't you want to check the output ´fixture->source´ ?

Reply:
This validates that the sdu, rtn and phy from the create parameters are correctly applied to the stream. Arguably there are more fields that could be verified, and I may add that later